client/src/components/Modals/AddGoalModal/AddGoalModal.jsx

```diff
@@ -18,50 +19,57 @@ const useStyles = makeStyles(({
   container: {
     display: 'flex',
     flexDirection: 'row',
-    padding: 10,
-    width: '390px',
-    height: '170px',
+    padding: 25,
+    width: 'fit-content',
+    height: 'fit-content',
     borderRadius: '8px',
-    fontFamily: 'Arial',
+    // fontFamily: 'Arial',
     fontSize: '18px',
     background: '#F2F2F2',
+    backgroundImage: 'url("https://selious.s3.amazonaws.com/selousSplice.PNG")',
     border: 2,
+    backgroundRepeat: 'no-repeat',
+    backgroundSize: '100% 20%',
     borderColor: '#9F6CB7',
     borderStyle: 'solid',
+    margin: '10% auto auto auto',
+    // fontFamily: "'Michroma', sans-serif !important",
```

Comment:
again please delete font family or update it to the correct font that Keenan specified in slack